github-dark.css

```diff
@@ -1659,6 +1659,10 @@
   .private .mini-repo-list-item {
     background: #221 !important;
   }
+  /*Private repo Lock color*/
+  .private a svg {
+		color: #ffda79 !important;
```

Comment:
I would use this selector to a existing yellow rule

e.g.

https://github.com/StylishThemes/GitHub-Dark/blob/5b619a13ff24012899dc66a85d8793268d43d83d/github-dark.css#L3431-L3433

Reply:
I think this color is too bright for here. I'm looking at using `#651` which is part of the generator and used in a few other spots. Just checking to see what else this may effect. 

![image](https://user-images.githubusercontent.com/831974/50983678-f1daa480-14cd-11e9-8244-ad3a48646b5b.png) ![image](https://user-images.githubusercontent.com/831974/50983216-00748c00-14cd-11e9-85e1-dc1af1f7ce03.png) ![image](https://user-images.githubusercontent.com/831974/50983234-0ec2a800-14cd-11e9-8897-c51baf0b7b48.png)

What's odd is they aren't styling this lock anywhere else (yet) and over the last year or so removed this color from the `private` label on repos.

![image](https://user-images.githubusercontent.com/831974/50983618-d4a5d600-14cd-11e9-9f8f-1814bf28c2c5.png)
![image](https://user-images.githubusercontent.com/831974/50983583-c061d900-14cd-11e9-8fcb-3405f8eea1ca.png)

https://github.com/StylishThemes/GitHub-Dark/blob/5b619a13ff24012899dc66a85d8793268d43d83d/github-dark.css#L3388-L3397

Reply:
So yea, the gold/yellow used on this lock is also used for PR check statuses. The red circle below on this PR can have pending/queued/etc. statuses which we're currently not handling (looks like we're not handling any of them actually). I'm working on an update now that should fix all of these.

Default color

![image](https://user-images.githubusercontent.com/831974/50984364-b8a33400-14cf-11e9-870f-67891f34a2d4.png)

Using `#cb4`

![image](https://user-images.githubusercontent.com/831974/50984404-cb1d6d80-14cf-11e9-9093-4af5185d1179.png)

Using `#651`

![image](https://user-images.githubusercontent.com/831974/50984587-41ba6b00-14d0-11e9-9020-2711656daf08.png)